config/network.go

```diff
@@ -31,10 +31,8 @@ const custom network = "custom"
 type GenesisFromJSON struct {
 	// L1: root hash of the genesis block
 	Root string `json:"root"`
-	// L1: block number in which the rollup was created
-	RollupCreationBlockNum uint64 `json:"rollupCreationBlockNumber"`
-	// L1: block number in which the rollup manager was created
-	RollupManagerCreationBlockNum uint64 `json:"rollupManagerCreationBlockNumber"`
```

Comment:
What happened with this parameter? We do not need it anymore? :slightly_smiling_face: